virttest/utils_test/libvirt.py

```diff
@@ -1511,7 +1511,11 @@ def check_state():
         return utils_misc.wait_for(check_state, timeout)
 
 
-def check_result(result, expected_fails=[], skip_if=[], any_error=False):
+def check_result(result,
```

Comment:
I noticed that you add one more parameter for check_result function, please make sure whether it will impact other use case where check_result were used.

Reply:
I think this may will not make impact. The new parameter has default value and i add it as the last parameters.